### PR TITLE
feat: ITV-26 add decoration pack page

### PR DESCRIPTION
**Pull request by LBHABEBIC**
# What?

added decoration page which dits between gas and laminate flooring. It renders a notes box and pva if the user clicks no, not delivered"

image

### CODE REVIEW

File: steps/PageSlugs.ts [hunk 1]
```diff
@@ -30,6 +30,7 @@ enum PageSlugs {
   AboutProperty = "about-property",
   PropertyOccupation = "property-occupation",
   Gas = "gas",
+  DecorationPack = "Decoration pack",
```
Comment (mrmartineau) [inline steps/PageSlugs.ts]: I think you also need to add a reference to the step in the `slugs` object in this file

**Review by mrmartineau — CHANGES_REQUESTED** [review]
looks great apart from the one thing I commented on

**Review by LBH-wgreeff — APPROVED** [review]
Looks good to me

**Review by mrmartineau — APPROVED** [review]
looks good to me

**Pull request merged**